recipes/iso3166/LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```

Comment:
You don't need to manually package the license since it is in the source distribution.